__tests__/app.test.js

```diff
@@ -232,4 +233,107 @@ describe("POST /api/reviews/:review_id/comments", () => {
+          category: "social deduction",
+          created_at: "2021-01-22T11:35:50.936Z",
+          votes: 8,
+        });
+      });
+  });
+  test("200: respond with review object with votes property correctly incremented - going into negatives", () => {
+    const votes = { inc_votes: -100 };
+    return request(app)
+      .patch("/api/reviews/6")
+      .send(votes)
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toEqual({
+          review_id: 6,
+          title: "Occaecat consequat officia in quis commodo.",
+          designer: "Ollie Tabooger",
+          owner: "mallionaire",
+          review_img_url:
+            "https://images.pexels.com/photos/278918/pexels-photo-278918.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+          review_body:
+            "Fugiat fugiat enim officia laborum quis. Aliquip laboris non nulla nostrud magna exercitation in ullamco aute laborum cillum nisi sint. Culpa excepteur aute cillum minim magna fugiat culpa adipisicing eiusmod laborum ipsum fugiat quis. Mollit consectetur amet sunt ex amet tempor magna consequat dolore cillum adipisicing. Proident est sunt amet ipsum magna proident fugiat deserunt mollit officia magna ea pariatur. Ullamco proident in nostrud pariatur. Minim consequat pariatur id pariatur adipisicing.",
+          category: "social deduction",
+          created_at: "2020-09-13T14:19:28.077Z",
+          votes: -92,
+        });
+      });
+  });
+  test("400: returns an error message when passed an invalid data type", () => {
+    const votes = { inc_votes: "banana" };
+    return request(app)
+      .patch("/api/reviews/5")
+      .send(votes)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid datatype found");
+      });
+  });
+  test("404: returns an error message when when passed correct data type but a review_id that does not exist", () => {
+    const votes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/reviews/10000")
+      .send(votes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Review ID does not exist");
+      });
+  });
+  test("400: return an error message when inc-votes is missing", () => {
+    const votes = { inc_votes: {} };
+    return request(app)
+      .patch("/api/reviews/5")
+      .send(votes)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid datatype found");
+      });
+  });

```

Comment:
I think this test is essentially the same as the one on line 309 - `inc_votes` is present on the request body but its value is invalid